core/lib/lantern/types/global.d.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './lantern.js';
+
+export as namespace Lantern;
```

Comment:
Global namespace is definitely convenient, but it will make it difficult to publish these types next to the source code in the future unless we do the `import * as Lantern from '../../types/lantern.js'` trick.

Since the types are already using local imports we could just update the existing import paths. Could save us some headache when publishing types in the future.